pkg/uroot/builder/binary.go

```diff
@@ -39,7 +39,7 @@ func (BinaryBuilder) Build(l ulog.Logger, af *initramfs.Files, opts Opts) error
 		wg.Add(1)
 		go func(p string) {
 			defer wg.Done()
-			result <- opts.Env.Build(
+			result <- opts.Env.BuildDir(
```

Comment:
do we have tests in uroot_test.go for the binary builder? if there aren't any, can you add a couple to make sure this still works?

Reply:
There was a test but it got disabled previously. I'll re-enable it once I got around the CircleCI issue.